tests/Unit/AcceleratorViewCopy/common.h

```diff
@@ -11,6 +11,12 @@
     printf ("error: TEST FAILED\n%s", KNRM );\
     abort();
 
+#define HostToDeviceCopyTest 0x1
+#define DeviceToDeviceCopyTest 0x2
+#define DeviceToHostCopyTest 0x4
+#define HostToDeviceAsyncCopyTest 0x18
+#define DeviceToHostAsyncCopyTest 0xFF
+
```

Comment:
It was a bug because it lost the ability to be able to run a particular test individually as I pointed out previously.  For example, after you switched back to the old hex values, now someone could run the DeviceToHostAsyncCopyTest alone without having to run all the previous tests.